src/app/zap-templates/zcl/data-model/chip/account-login-cluster.xml

```diff
@@ -36,6 +36,10 @@ limitations under the License.
       <arg name="setupPIN" type="CHAR_STRING"/>
     </command>
 
+    <command source="client" code="0x02" name="Logout" optional="false">
```

Comment:
The `code` does not match the spec.  Why not?  We should match the spec.  That sees to be true of a lot of the `code` values in this file.